# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1636752042 -3600
# Parent  22783f61dfa67858945be6c044ab4fd2eb3321e9
Backed out changeset 9bd7b7700df2 (bug 1669063)

This is a partial backout. Non-offending changes were left in place and the test changes weren't backed out.

diff --git a/mail/base/content/msgHdrView.js b/mail/base/content/msgHdrView.js
--- a/mail/base/content/msgHdrView.js
+++ b/mail/base/content/msgHdrView.js
@@ -347,18 +347,16 @@ function OnResizeExpandedHeaderView() {
     document
       .getElementById("expandedHeaderView")
       .setAttribute(
         "height",
         document.getElementById("expandedHeadersTopBox").clientHeight +
           document.getElementById("expandedHeaders2").clientHeight
       );
   }
-
-  onHeaderResize();
 }
 
 var MsgHdrViewObserver = {
   observe(subject, topic, prefName) {
     // verify that we're changing the mail pane config pref
     if (topic == "nsPref:changed") {
       if (prefName == "mail.showCondensedAddresses") {
         gShowCondensedEmailAddresses = Services.prefs.getBoolPref(
@@ -1261,49 +1259,35 @@ function ClearCurrentHeaders() {
   currentHeaderData = {};
   // eslint-disable-next-line no-global-assign
   currentAttachments = [];
 }
 
 function ShowMessageHeaderPane() {
   document.getElementById("msgHeaderView").collapsed = false;
   document.getElementById("mail-notification-top").collapsed = false;
-  onHeaderResize();
 }
 
 function HideMessageHeaderPane() {
-  let header = document.getElementById("msgHeaderView");
-  header.collapsed = true;
+  document.getElementById("msgHeaderView").collapsed = true;
   document.getElementById("mail-notification-top").collapsed = true;
 
   // Disable the Message/Attachments menuitem.
   document.getElementById("msgAttachmentMenu").setAttribute("disabled", "true");
 
   // Disable the app menu attachment menu in there as well.
   document
     .getElementById("appmenu_msgAttachmentMenu")
     ?.setAttribute("disabled", "true");
 
   // disable the attachment box
   document.getElementById("attachmentView").collapsed = true;
   document.getElementById("attachment-splitter").collapsed = true;
 
   gMessageNotificationBar.clearMsgNotifications();
-
-  // Always remove the shrink attribute in order to start with the right size
-  // when loading a new message.
-  header.removeAttribute("shrink");
-}
-
-function onHeaderResize() {
-  let header = document.getElementById("msgHeaderView");
-  // Arbitrary size of 700px, which should be safe for now. This is a temporary
-  // fix while we wait for the @container media query to be supported, and in
-  // preparation for the full message header rebuild.
-  header.toggleAttribute("shrink", header.getBoundingClientRect().width < 700);
 }
 
 /**
  * Take string of newsgroups separated by commas, split it
  * into newsgroups and send them to the corresponding
  * mail-newsgroups-headerfield element.
  *
  * @param headerEntry  the entry data structure for this header
diff --git a/mail/themes/shared/mail/messageHeader.css b/mail/themes/shared/mail/messageHeader.css
--- a/mail/themes/shared/mail/messageHeader.css
+++ b/mail/themes/shared/mail/messageHeader.css
@@ -76,24 +76,16 @@
 .toolbarbutton-menubutton-dropmarker::before {
   background-image: none;
 }
 
 .msgHeaderView-button .toolbarbutton-text {
   padding-inline-start: 2px;
 }
 
-#msgHeaderView[shrink] .msgHeaderView-button .toolbarbutton-text {
-  display: none;
-}
-
-#msgHeaderView[shrink] .toolbarbutton-1 .toolbarbutton-menu-dropmarker {
-  margin-inline: 3px;
-}
-
 .hdrReplyToSenderButton,
 .hdrDummyReplyButton,
 .hdrReplyButton {
   list-style-image: url("chrome://messenger/skin/icons/reply.svg");
 }
 
 .hdrReplyAllButton {
   list-style-image: url("chrome://messenger/skin/icons/replyall.svg");
@@ -243,17 +235,16 @@
 /* ::::: msg header captions ::::: */
 
 #header-view-toolbar .msgHeaderView-button[disabled="true"] {
   display: none;
 }
 
 #header-view-toolbar {
   -moz-box-pack: end;
-  margin-inline-end: 3px;
 }
 
 .headerName {
   padding: 0;
   margin-inline-end: 3px;
 }
 
 .headerValue {
